examples/siro_sandbox/sandbox_app.py

```diff
@@ -883,7 +887,8 @@ def parse_debug_third_person(args, framebuffer_size):
         debug_third_person_height,
     ) = parse_debug_third_person(args, framebuffer_size)
 
-    config = habitat.get_config(args.cfg, args.cfg_opts)
+    config = get_baselines_config(args.cfg, args.cfg_opts)
```

Comment:
Use baselines config instead of benchmark config. Baselines config should contain all the information not only about the benchmark but also about baseline agent that we should use to configure controllers.